<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuidSimpleDevice</class>
 <widget class="QWidget" name="GuidSimpleDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Please obtain the following robot parameters for configuration according to the product specification or design drawings. Click next after completion.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Battery detection type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="customComboBox" name="comboBox_batteryType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>gyroscop type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="customComboBox" name="comboBox_gyroType"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>deriver brand:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="customComboBox" name="comboBox_driverBrand"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customComboBox</class>
   <extends>QComboBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
